package.json

```diff
@@ -28,43 +28,59 @@
   },
   "homepage": "https://github.com/tensorflow/tensorboard#readme",
   "devDependencies": {
-    "@angular/bazel": "^8.2.14",
-    "@angular/cli": "^8.3.21",
-    "@angular/compiler": "^8.2.14",
-    "@angular/compiler-cli": "^8.2.14",
-    "@bazel/bazel": "0.26.1",
-    "@bazel/hide-bazel-files": "^0.34.0",
+    "@angular/bazel": "^9.0.0",
+    "@angular/cli": "^9.0.1",
+    "@angular/compiler": "^9.0.0",
+    "@angular/compiler-cli": "^9.0.0",
+    "@bazel/bazel": "^0.23.2",
     "@bazel/ibazel": "^0.10.3",
-    "@bazel/jasmine": "^0.32.2",
-    "@bazel/karma": "^0.36.1",
-    "@bazel/typescript": "^0.34.0",
+    "@bazel/jasmine": "^1.2.4",
```

Comment:
These packages are all provided by rules_nodejs and in the past we had them on the same version as our WORKSPACE `rules_nodejs` dep (which this PR sets to 1.6.0).

Would it be possible to set these all to `^1.6.0` so that we're matching the two types of dependencies?